### ISSUE TITLE
Missing .env File and Exclusion of Folder in Production Build

### ISSUE THREAD

**Issue by aalistarbid** [statbid/shopping-feed-analyzer#3]
We need a .env file to define our base URL and other environment variables for easy configuration across different environments.

During the build process,  folder (e.g., dictionaries) is not being included in the production build output. This is causing path issues when running the application in production, as these resources are required. We can run through "npm run dev" locally but for production we need to point directly to build.

- Modify the build configuration to ensure the required folders, like dictionaries, are included in the production build. This may involve setting the folder path at the root level or adjusting the build script to copy these folders correctly.

**Comment by DGuzmanRomano** [comment]
I've completed the setup for the .env configuration and adjusted the build process to include necessary folders like dictionaries in the production output. I created a .env.example file as a template for required environment variables, which other developers can use to set up their own .env files.

The application should now work seamlessly across different environments, both in development and production. Let me know if there’s anything else you’d like me to address.
